server/services/communityService.ts

```diff
@@ -33,14 +34,18 @@ export default class CommunityService {
     return this.communityApiClient.getTeamCaseload(teamCodes)
   }
 
+  async getAnOffendersManagers(crn: string): Promise<CommunityApiOffenderManager[]> {
+    return this.communityApiClient.getAnOffendersManagers(crn)
+  }
+
   async searchProbationers(searchCriteria: SearchDto): Promise<OffenderDetail[]> {
     return this.probationSearchApiClient.searchProbationer(searchCriteria)
   }
 
-  async getProbationer(nomisId: string): Promise<OffenderDetail> {
-    const deliusRecords = await this.probationSearchApiClient.searchProbationer({ nomsNumber: nomisId })
+  async getProbationer(searchDto: SearchDto): Promise<OffenderDetail> {
+    const deliusRecords = await this.probationSearchApiClient.searchProbationer(searchDto)
```

Comment:
Not sure why the change of parameter type? Do we search here by different values?

Reply:
No reason, I was going to use the endpoint to search for offenders by CRN but didn't need to in the end